Bottom: 726b71a4a9cefdea458fd06806a97893ee48c18d
Top:    c9bffef4aed312a07adc1f19bc92101694dcc7f8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 01:00:46 +0100

Refresh of w-c-split-gathering-from-real-

---

diff --git a/src/draw.c b/src/draw.c
index 7aeaa69..b9a812e 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -237,7 +237,7 @@ draw_element_pins_and_pads (ElementType *element)
   END_LOOP;
   PIN_LOOP (element);
   {
-    DrawPin (pin, true);
+    DrawPin (pin);
   }
   END_LOOP;
 }
